src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/Service.tsx

```diff
@@ -30,32 +30,55 @@ import { isUpdated } from '../../Configurations.utils'
 import ChangeSourceTableContainer from './components/ChangeSourceTableContainer/ChangeSourceTableContainer'
 import css from './Service.module.scss'
 
-function Service({
-  value: initialValues,
-  onSuccess,
-  cachedInitialValues,
-  setDBData,
-  onDiscard,
-  serviceTabformRef,
-  onChangeMonitoredServiceType,
-  isTemplate,
-  updateTemplate
-}: {
-  value: MonitoredServiceForm
-  onSuccess: (val: any) => Promise<void>
-  cachedInitialValues?: MonitoredServiceForm | null
-  setDBData?: (val: MonitoredServiceForm) => void
-  onDiscard?: () => void
-  serviceTabformRef?: any
-  onChangeMonitoredServiceType: (updatedValues: MonitoredServiceForm) => void
-  isTemplate?: boolean
-  updateTemplate?: (template: NGTemplateInfoConfigWithMonitoredService) => Promise<void>
-}): JSX.Element {
+function Service(
+  {
+    value: initialValues,
+    onSuccess,
+    cachedInitialValues,
+    setDBData,
+    onDiscard,
+    serviceTabformRef,
+    onChangeMonitoredServiceType,
+    isTemplate,
+    updateTemplate
+  }: {
+    value: MonitoredServiceForm
+    onSuccess: (val: any) => Promise<void>
+    cachedInitialValues?: MonitoredServiceForm | null
+    setDBData?: (val: MonitoredServiceForm) => void
+    onDiscard?: () => void
+    serviceTabformRef?: any
+    onChangeMonitoredServiceType: (updatedValues: MonitoredServiceForm) => void
+    isTemplate?: boolean
+    updateTemplate?: (template: NGTemplateInfoConfigWithMonitoredService) => Promise<void>
+  },
+  formikRef?: any
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9306981)
```suggestion
  formikRef?: unknown
```